### PR TITLE
RANCHER-1342 PR for CodeReview

### PR DESCRIPTION
**Pull request by eldiiar-duishenaliev**

### CODE REVIEW

File: charts/folio-common/templates/_deployment.tpl [hunk 1]
```diff
@@ -76,6 +107,11 @@ spec:
               containerPort: {{ .Values.jmx.port | default "1099" }}
               protocol: "TCP"
             {{- end }}
+            {{- if .Values.eureka.enabled }}
+            - name: "sidecar"
+              containerPort: "8082"
```
Comment (OHaimanov) [inline charts/folio-common/templates/_deployment.tpl:112]: Variable with default val

File: charts/folio-common/templates/_deployment.tpl [hunk 2]
```diff
@@ -65,6 +65,37 @@ spec:
...
+          - name: MODULE_NAME
+            value: {{ .Chart.Name | quote }}
+          - name: SIDECAR_FORWARD_UNKNOWN_REQUESTS
+            value: "true"
+          - name: SIDECAR_URL
+            value: "http://{{ .Chart.Name }}:8082"
+          - name: SIDECAR
+            value: "true"
+          - name: JAVA_OPTS
+            value: "--server.port=8082 -Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager -XX:+UseZGC -Xmx128m"
```
Comment (OHaimanov) [inline charts/folio-common/templates/_deployment.tpl:90]: Port as Variable with default val

File: charts/folio-common/templates/_deployment.tpl [hunk 3]
```diff
@@ -65,6 +65,37 @@ spec:
...
+          - name: TM_CLIENT_URL
+            value: "http://mgr-tenants"
+          - name: MODULE_URL
+            value: "http://{{ .Chart.Name }}"
+          - name: MODULE_NAME
+            value: {{ .Chart.Name | quote }}
+          - name: SIDECAR_FORWARD_UNKNOWN_REQUESTS
+            value: "true"
+          - name: SIDECAR_URL
+            value: "http://{{ .Chart.Name }}:8082"
```
Comment (OHaimanov) [inline charts/folio-common/templates/_deployment.tpl:86]: Port as Variable with default val

File: charts/folio-common/templates/_service.tpl [hunk 4]
```diff
@@ -13,6 +13,12 @@ spec:
       port: {{ .port }}
       targetPort: {{ .targetPort }}
     {{- end }}
+    {{- if .Values.eureka.enabled }}
+    - name: sidecar
+      protocol: TCP
+      port: 8082
```
Comment (OHaimanov) [inline charts/folio-common/templates/_service.tpl:19]: Variable with default val

**Review by yaroslavishchenko — APPROVED** [review]

**Pull request closed without merging**